OrderApi/Migrations/20180816230225_default.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
```

Comment:
I dont think we need this, I believe we are supposed to create our own migration file for our own project rather than using the one she had.